test/archivespec.js

```diff
@@ -187,4 +187,38 @@ describe("archive", function () {
       });
     });
   });
+  describe('download_folder', function(){
+    this.timeout(TIMEOUT.LONG);
+    uploader.upload(`folder/${PUBLIC_ID1}`)
+      .then((uploadResult)=> {
+        expect(uploadResult).not.to.be(null);
+      });
+
+    after(function () {
+      return cloudinary.api.delete_resources_by_prefix('folder/')
+        .then((deleteResult)=> {
+          expect(deleteResult).not.to.be(null);
+        });
+    });
```

Comment:
Why do we need a setup and teardown if we're just checking URLs?

Reply:
I wanted to test it on a real URL but I guess it doesn't matter. Removed.